jest.setup.js

```
@@ -0,0 +1,5 @@
+// Zero safe-area insets without needing a provider in every test; a minimal
+// inline mock is all these tests consume from the library.
+jest.mock("react-native-safe-area-context", () => ({
```

ESLint

[error] 'jest' is not defined.

(no-undef)